eth/zz_deepcopy_generated.go

```diff
@@ -0,0 +1,606 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Kind of weird the tool includes this license in the auto-generated code, going to open an issue upstream to see if there's a way around that.

Reply:
Fixed in e4d83d7 by using `dev/null` as the boilerplate.

Reply:
the _best_ boilerplate.